doc/classes/InputEventAction.xml

```diff
@@ -16,6 +16,9 @@
 		<member name="action" type="StringName" setter="set_action" getter="get_action" default="&amp;&quot;&quot;">
 			The action's name. Actions are accessed via this [String].
 		</member>
+		<member name="event_id" type="int" setter="set_event_id" getter="get_event_id" default="-1">
+			The real event index in action this event corresponds to (from events defined for this action in the [InputMap]). If [code]-1[/code], a unique ID will be used and actions pressed with this ID will need to be released with another [InputEventAction].
```

Comment:
Does the ID matter for any checks like pressing/releasing tracking, and if so does it have to match in some special way? If so details on possible issues with using `-1` should be detailed

Reply:
But if you then pass another event manually with the same ID, is it released?

Reply:
It depends if the event has `pressed` enabled. Multiple pressed events will not release action, you need a release event.

Reply:
So if you pass a release event? I said "passed" not "pressed", assumed it to be `pressed == false` as normal conditions, the difference being the id

Reply:
Action pressed state is tracked by event index. If the action is pressed, it needs a release event with the same ID as the pressed event to get released. Action remains pressed as long as at least one ID/index was not released.

Reply:
I think the wording then needs to be clarified as from the current wording it sounds like it's sufficient to pass another event with `-1`, which I assume won't be the case here as it's based on the size?

Reply:
Well yeah, if between press and release you add a new event in the InputMap, the action won't be released using `-1`. But it's such an unrealistic corner case that I'm not sure if it's even worth mentioning 🤔

Reply:
Well the description says "need to be released with another [InputEventAction]." in the same sentence as talking about the id, so I think it'll cause confusion, but idk